velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -221,6 +222,11 @@ class HiveConnectorSplitBuilder {
   }
 
  private:
+  static std::string fixUpPath(const std::string& path) {
```

Comment:
can you add a comment here. Why do we need this?

Reply:
Please open a separate PR for this as the changes are unrelated.

Reply:
This needs tests as well.

Reply:
So, I have no idea why originally the string "file:" was prepended. I suspect the UT was not being generic. I can restate in english what the method is doing. Is that ok?

Reply:
> Please open a separate PR for this as the changes are unrelated.

Without this change the TpchBenchmark cannot support S3.

Reply:
> This needs tests as well.

Am I to understand this test file needs tests?

Reply:
Since CMake changes are being handled separately, this can stay in this PR.